### DISCUSSION TITLE
Tag editing with Kid3 seems to be gone

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1444] [Fixed Issues]
@rern 
In the actual release ID3 tag-editing with Kid3 is no more possible. What's the reason?

**Comment by rern** [comment]
Tags are normally based on each directory/folder.
It's applicable in file modes only. (NAS, SD and USB)

**Reply by klassikmann** [reply]
I can't find the corresponding menu entry:
![Screenshot](https://github.com/rern/rAudio/assets/25383513/5260493f-06b1-4d72-bd04-ee5ddec16300)

**Reply by R101G** [reply]
@klassikmann 

That's not a menu I can see anywhere. I am on 20231001 from updates rather than a new image, and the tag editor is present in the library file menu.

**Comment by klassikmann** [comment]
@R101G 
Interesting. Please have a look into https://github.com/rern/rAudio/discussions/1438
After the problems 1) with an update of a working installation and 2) a new sd-card based on the last image (i20230918). I decided to make a DIY-system according https://github.com/rern/rOS; no problems at all but I learned now that  Kid3 tag-editing is missing.
Now I have the suspicion that there is something wrong with the build framework of rAudio.

**Reply by R101G** [reply]
@klassikmann 
I have not experienced any of the problems in #1438. The system operation is definitely more clunky than it used to be, though.

**Comment by ceejayemm** [comment]
@klassikmann 
This is fixed for me in 20231013

Chris
